src/components/ArrowLeftIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import type { SVGProps } from "react"
+
+export function ArrowLeftIcon(props: SVGProps<SVGSVGElement>) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      stroke-width="1.5"
+      stroke-linecap="round"
+      stroke-linejoin="round"
```

Comment:
```suggestion
      strokeWidth="1.5"
      strokeLinecap="round"
      strokeLinejoin="round"
```

logにエラーが出ていたので、JSXの書き方に修正する必要がありそうです！

Reply:
凡ミスですね、すみません💦
(VSCodeでデフォルトで検出されなかったのなんでだろう🤔)